team-manager/src/main/java/ua/knu/knudev/teammanager/service/requirementsProcessor/MasterSpecification.java

```diff
@@ -0,0 +1,14 @@
+package ua.knu.knudev.teammanager.service.requirementsProcessor;
+
+import ua.knu.knudev.teammanager.requirements.MasterDeveloperRequirements;
+
+public class MasterSpecification implements Specification<MasterDeveloperRequirements> {
+
+    @Override
+    public boolean isSatisfiedBy(MasterDeveloperRequirements masterDeveloperRequirements) {
+        return masterDeveloperRequirements.getPreMasterDurationInMonths() > 12
+                && masterDeveloperRequirements.getManagedStudentsInPreCampus() > 2
+                && masterDeveloperRequirements.getPreMasterProjectsCount() > 0
+                && masterDeveloperRequirements.getTechLeadScore() >= 7;
+    }
```

Comment:
And also extract this all values into app config

Reply:
all correct comments all improved all code was remaded